### PR TITLE
Move project details to settings

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Project details page (label, id, created date, organism, etc...) was just hanging out with the other tabs and did not match the feel that we are working towards.  I moved it into the setting panel, utilized ant design and match the style of the analysis details.

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [x] Tests added (or description of how to test) for any new features.
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/webapp/resources/js/pages/projects/details/ProjectDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+  /**
+   * When a field is updated, submitted it to the server to be saved.
+   * Display a notification on the result of the update, and then update the
+   * UI accordingly.
+   *
+   * @param {string} field to be updated
+   * @param {string} value to be store in the field
+   */
+  const updateField = (field, value) => {
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/details/ProjectDetails.jsx:59]: Probably add a check to make sure a value was actually changed and only make a server call if there was a change detected

File: src/main/webapp/pages/projects/settings/pages/details.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  data-layout-decorate="~{projects/settings/_base}"
+>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Title</title>
+    <link rel="stylesheet" href="/dist/css/project-details.bundle.css">
+  </head>
+  <body>
+    <div id="details-root" layout:fragment="settings-content" >
+      <div>LOADING ...</div>
```
Comment (deepsidhu85) [inline src/main/webapp/pages/projects/settings/pages/details.html:15]: Internationalize the text

File: src/main/webapp/resources/js/pages/projects/details/ProjectDetails.jsx (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/details/ProjectDetails.jsx:59]: Updated in c6f2992620113f18985e05705da2089dbff7745a, also added a check to ensure that the project label met the requirements for length.

File: src/main/webapp/pages/projects/settings/pages/details.html (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/pages/projects/settings/pages/details.html]: Updated in dea2b0460f709bf2d12e16db08482f38c7a98642

File: src/main/webapp/resources/js/pages/projects/details/ProjectDetails.jsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+import { BasicList } from "../../../components/lists";
+import { formatInternationalizedDateTime } from "../../../utilities/date-utilities";
+import { EditableParagraph } from "../../../components/ant.design";
+import { OntologySelect } from "../../../components/ontology";
+import { TAXONOMY } from "../../../apis/ontology/taxonomy";
+
+const { Paragraph, Title } = Typography;
+
+/**
+ * Reducer for the state of the project details.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/details/ProjectDetails.jsx:16]: Context really only helps if you are doing a lot of props drilling (passing props down through a lot of components), it won't help in this situation.  Eventually we might need it if you move all code encapsulating a project into a "single page react app" but we can reconsider that when we get there.

[the remark this answers is not in the document]
Comment (deepsidhu85) [reply src/main/webapp/resources/js/pages/projects/details/ProjectDetails.jsx:16]: Sounds good. Thanks!

**Pull request merged**